ruby_connectors/spec/temporal/activities/write_database_data_activity_spec.rb

```
@@ -0,0 +1,154 @@
+
+    context "when error occurs" do
+      before do
+        allow(client).to receive(:write)
+          .and_raise(StandardError.new("Write failed"))
+      end
+
+      it "returns error status with message" do
+        result = activity.execute(params)
+
+        expect(result).to eq({
+          status: "error",
```

rubocop (1.69.1)

[convention] Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.

(Layout/FirstHashElementIndentation)